src/ray/core_worker/lib/java/io_ray_runtime_object_NativeObjectStore.cc

```diff
@@ -22,10 +22,15 @@
 #include "ray/core_worker/core_worker.h"
 
 ray::Status PutSerializedObject(JNIEnv *env, jobject obj, ray::ObjectID object_id,
-                                ray::ObjectID *out_object_id, bool pin_object = true) {
+                                ray::ObjectID *out_object_id, bool pin_object = true,
+                                const std::string *address = nullptr) {
```

Comment:
```suggestion
                                const std::string *serialized_owner_address = nullptr) {
```

Reply:
Also, there's no need to use a pointer here. Just use an empty string to represent the no assignment behavior.